middleware/authenticate.js

```diff
@@ -1,20 +1,31 @@
-// middleware/authenticate.js
-
 const jwt = require('jsonwebtoken');
 
 const authenticate = (req, res, next) => {
-  const token = req.headers['authorization'] && req.headers['authorization'].split(' ')[1];
-  if (!token) {
-    return res.status(401).json({ message: 'No token provided' });
-  }
-  jwt.verify(token, process.env.JWT_SECRET, (err, decoded) => {
-    if (err) {
-      return res.status(401).json({ message: 'Invalid or expired token' });
+  try {
+    // Check if the 'Authorization' header is present and starts with 'Bearer' (case-insensitive)
+    const authHeader = req.headers['authorization'];
+    if (!authHeader || !authHeader.toLowerCase().startsWith('bearer ')) {
+      return res.status(401).json({ message: 'Authorization header must be in Bearer token format' });
```

Comment:
"Bearer token format" shouldn't be mentioned in a log. It discloses too much information about the security of the application.

Reply:
Issues is resolved and token stored in Encrypted format